CsvParser/csvParser.go

```diff
@@ -129,3 +143,140 @@ func makeJsonObjFile(arr []incident.Incident) *os.File {
 
 	return f
 }
+
+//convert a default incident that implements our interface to a less safe incident type to be dumped on .json file
```

Comment:
What does a "less safe incident type" mean? Please change phrasing in the comment